docs/user-guide/src/irso/introduction.md

```diff
@@ -0,0 +1,72 @@
+# Ironic Standalone Operator
+
+Ironic Standalone Operator (IrSO) is a Kubernetes controller that installs and
+manages Ironic in a configuration suitable for Metal3. IrSO has the following
+features:
+
+- Flexible networking configuration, support for Keepalived.
+- Using SQLite or MariaDB as the database backend.
+- Optional support for a DHCP service (dnsmasq).
+- Optional support for automatically downloading an
+  [IPA](../ironic/ironic-python-agent.md) image.
+
+IrSO uses [ironic-image](../ironic/ironic-container-images.md) under the hood.
+
+## Installing Ironic Standalone Operator
+
+On every source code change, a new IrSO image is built and published at
+`quay.io/metal3-io/ironic-standalone-operator`. To install it in your cluster,
+you can use the Kustomize templates provided in the source repository:
+
+```console
+git clone https://github.com/metal3-io/ironic-standalone-operator
+cd ironic-standalone-operator
+git checkout -b <DESIRED BRANCH OR main>
+
+make install deploy
```

Comment:
In the future we should start publishing the output of `kustomize build` as a release artifact so users can just do a `kubectl apply -f [URL]`. This is fine for now though, no need to change!

Reply:
Yeah, I totally agree. Do we already do it for CAPM3?

Reply:
Yes, it is part of how CAPI expects things to work. So it is published as `infrastructure-components.yaml` on every release. That allows `clusterctl init --infrastructure=metal3` to automatically find the latest and install it :slightly_smiling_face: